### PR TITLE
Add additional devices and proofs to the list

### PR DESCRIPTION
**Pull request by friissoren**
Signed-off-by: Soren Friis <soren.friis@intel.com>

### CODE REVIEW

File: supported-smartphones.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ The easiest way to verify support is to read the Android Feature flags.
 
 #### I have a phone that isn’t listed and I want to add it.
 
-That’s great! Please follow the methodology to verify its capabilities and create a Pull Request with additional information and screenshots from the tests proving the support. 
+That’s great! Please follow the methodology to verify its capabilities and create a Pull Request with additional information and screenshots from the tests proving the support.
 
 For the Pull Request, please upload the screenshots to the `receiver_proofs/xxx_yyy` folder in the repository where `xxx` is the lowercase smartphone manufacturer (e.g. samsung) and `yyy` is the lowercase smartphone model (e.g. `galaxy_note_10`). Name the screenshots in the folder accordingly to tested capabilities. We recommend to use the following template:
```
Comment (lukasbrchl) [inline supported-smartphones.md:102]: I see that you changed my intention about using lowercase characters. I am fine with it, but could you please edit this description before we merge it? :)

Comment (friissoren) [reply supported-smartphones.md:102]: Sorry, I completely missed that. I will update the instructions. It seems more readable to me when you use both lower and upper case letters.

Comment (lukasbrchl) [reply supported-smartphones.md:102]: I agree that it is more readable. But also a little harder to write correctly which might make it inconsistent in the future. But let's keep it your way now and then we can fine-tune it

**Comment by lukasbrchl** [comment]
Great to see more proofs! I checked the links and everything seems to be valid

**Pull request merged**